README.md

```diff
@@ -1,15 +1,103 @@
-# react-lib-template
-A template that aims to make the implementation of `React` component packages easier and more methodic.
+# react-icon
+
+[![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][build-status-image]][build-status-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
+
+[npm-url]:https://npmjs.org/package/@moxy/react-icon
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/react-icon.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/react-icon.svg
+[build-status-url]:https://github.com/moxystudio/react-icon/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/react-icon/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/react-icon
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-icon/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-icon
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-icon.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-icon?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-icon.svg
+
+A component to render `SVG` icons.
+
+## Installation
+
+```sh
+$ npm install @moxy/react-icon
+```
+
+This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
+
+## Motivation
+
+This package was implemented to help import `SVG` icons in `react` projects.
 
 ## Usage
 
-This repo serves as the template for the creation of MOXY's base `React` components. To use this, just select `@moxystudio/react-lib-template` as the template when creating a new repo for your new package, and you're all set to start working.
+Create a `react-icons` folder on your `shared/modules`, and export icons from there.
```

Comment:
Now that I'm thinking about it... Should this be the example? This seems very related with our `next-with-moxy` 🤔 

Should we have an example that would work in almost any react project?